packages/airnode-deployer/terraform/airnode/aws/templates/httpSignedDataApiGateway.yaml.tpl

```diff
@@ -35,10 +35,7 @@ components:
 
     EndpointRequestExample:
       summary: Endpoint request example
-      value: { "parameters": { "from": "EUR", "amount": 5 } }
-    EndpointResponseExample:
```

Comment:
Why was the response example removed? Just curious

Reply:
Just out of convenience. We don't have these in the GCP version (they're using old swagger format so we can't), it's constantly outdated and it doesn't show anywhere in AWS.